src/main/java/edu/kis/powp/jobs2d/command/ComplexCommand.java

```diff
@@ -9,10 +9,17 @@ public class ComplexCommand implements ICompoundCommand {
 
     private final List<DriverCommand> commandList;
 
+    String name=null;
```

Comment:
should be private!
I suggest avoiding this null value. Is public ComplexCommand(List<DriverCommand> commandList) constructor necessary?